docs/api/io/BufferedSerial.md

```diff
@@ -0,0 +1,73 @@
+# BufferedSerial
+
+<span class="images">![](https://os.mbed.com/docs/development/mbed-os-api-doxy/classmbed_1_1_buffered_serial.png)<span>`BufferedSerial` class hierarchy</span></span>
+
+The `BufferedSerial` class provides UART functionality. We recommend you use this class for serial data transfers. You can use it to send and receive bytes of data in a sequence using separate transmit (TX) and receive pins (RX). Communication can be between two processors or for sending text to a console.
```

Comment:
What does this last sentence mean?

Reply:
It means that a device can interface with another device (such as sensors, or another board) to exchange data or to send text to be displayed on a text-based computer interface.

Reply:
I would either remove the original sentence or replace it with the explanation in your comment.

Reply:
Thanks. [Done](https://github.com/ARMmbed/mbed-os-5-docs/pull/1220/commits/0c90ed33cc75aadc0b1a82485ffd0ba90316daa1).